readthedocs/api/v3/serializers.py

```diff
@@ -368,6 +383,16 @@ def get_downloads(self, obj):
 
         return data
 
+    def get_aliases(self, obj):
+        if obj.slug in (STABLE, LATEST):
```

Comment:
Aliases are only valid for machine created versions.

```suggestion
        if obj.machine and obj.slug in (STABLE, LATEST):
```